aqua/ShiftIt/files/patch-ShiftIt-ShiftIt.xcodeproj-project.pbxproj.diff

```diff
@@ -1,35 +1,36 @@
---- ShiftIt/ShiftIt.xcodeproj/project.pbxproj	2014-12-08 18:06:57.000000000 -0500
-+++ ShiftIt/ShiftIt.xcodeproj/project.pbxproj	2014-12-08 18:10:02.000000000 -0500
-@@ -853,7 +853,7 @@
- 				GCC_WARN_UNUSED_PARAMETER = NO;
- 				GCC_WARN_UNUSED_VARIABLE = YES;
+diff --git a/ShiftIt/ShiftIt.xcodeproj/project.pbxproj b/ShiftIt/ShiftIt.xcodeproj/project.pbxproj
+index 66908fc..c900ad5 100644
+--- ShiftIt/ShiftIt.xcodeproj/project.pbxproj
++++ ShiftIt/ShiftIt.xcodeproj/project.pbxproj
+@@ -1013,13 +1013,13 @@
  				HEADER_SEARCH_PATHS = (
+ 					/opt/local/include,
+ 					/opt/X11/include,
```

Comment:
Looks like the developers changed their Xcode project and now /opt/local/include and /opt/X11/include are automatically searched. In MacPorts, we don't want that; we only want to search `@PREFIX@/include`. Could you adjust the patch to remove those unwanted paths everywhere they occur? Same for /opt/local/lib and /opt/X11/lib where we only want `@PREFIX@/lib`.